rest_framework/views.py

```diff
@@ -108,6 +110,14 @@ class APIView(View):
 
     schema = DefaultSchema()
 
+    def __init__(self, **kwargs):
+        super(APIView, self).__init__(**kwargs)
+        if self.schema is not None:
+            # copy class-level schema to prevent instances using the same object
+            if 'schema' not in self.__dict__:
+                self.schema = deepcopy(self.schema)
+            self.schema.view = self
```

Comment:
Can we do this in `ViewInspector.__get__()`?

Reply:
Let me take a second look at this. It might be possible to do this with `__set__`

Reply:
Alright, updated the descriptor with a setter. This seems to work fine.